inspire_disambiguation/api.py

```diff
@@ -47,11 +49,22 @@ def train_and_save_ethnicity_model(load_data_path, save_model_path):
         load_data_path (str): Full path to training data for ethnicity estimator.
         save_model_path (str): Full path where trained ethnicity model will be saved.
     """
+    _start = datetime.now()
     estimator = EthnicityEstimator()
     estimator.load_data(load_data_path)
+    _load_time = datetime.now()
     LOGGER.info("Training EthnicityEstimator. May take a while...")
     estimator.fit()
+    _training_time = datetime.now()
     estimator.save_model(save_model_path)
+    _save_time = datetime.now()
+    LOGGER.info(
+        "Training ethnicity model",
+        load_data_runtime=str(_load_time - _start),
+        training_model_runtime=str(_training_time - _load_time),
+        save_model_runtime=str(_save_time - _training_time),
```

Comment:
I don't think all these runtimes are relevant. I think only the total runtime is relevant. Those may be helpful as well in case we want to see where it is inefficient, but I still feel this is too much

Reply:
Discussed with Micha.